### ISSUE TITLE
Getting error during building project as 'GoThreeBars' is not exported from 'react-icons/go' 

### ISSUE THREAD

**Issue by VishalSingh4310** [themesberg/flowbite-react#818]
**### Getting this error few hours ago** 

**Attempted import error: 'GoThreeBars' is not exported from 'react-icons/go' (imported as 'GoThreeBars').**
ERROR in ./node_modules/flowbite-react/lib/esm/components/Navbar/NavbarToggle.js 9:23-34

export 'GoThreeBars' (imported as 'GoThreeBars') was not found in 'react-icons/go' (possible exports: GoAlert, GoAlertFill, GoArchive, GoArrowBoth, GoArrowDown, GoArrowDownLeft, GoArrowDownRight, GoArrowLeft, GoArrowRight, GoArrowSwitch, GoArrowUp, GoArrowUpLeft, GoArrowUpRight, GoBeaker, GoBell, GoBellFill, GoBellSlash, GoBlocked, GoBold, GoBook, GoBookmark, GoBookmarkFill, GoBookmarkSlash, GoBookmarkSlashFill, GoBriefcase, GoBroadcast, GoBrowser, GoBug, GoCalendar, GoCheck, GoCheckCircle, GoCheckCircleFill, GoCheckbox, GoChecklist, GoChevronDown, GoChevronLeft, GoChevronRight, GoChevronUp, GoCircle, GoCircleSlash, GoClock, GoClockFill, GoCloud, GoCloudOffline, GoCode, GoCodeOfConduct, GoCodeReview, GoCodeSquare, GoCodescan, GoCodescanCheckmark, GoCodespaces, GoColumns, GoCommandPalette, GoComment, GoCommentDiscussion, GoCommit, GoContainer, GoCopilot, GoCopy, GoCpu, GoCreditCard, GoCrossReference, GoDash, GoDatabase, GoDependabot, GoDesktopDownload, GoDeviceCameraVideo, GoDeviceDesktop, GoDeviceMobile, GoDiamond, GoDiff, GoDiscussionClosed, GoDiscussionDuplicate, GoDiscussionOutdated, GoDot, GoDotFill, GoDownload, GoDuplicate, GoEye, GoEyeClosed, GoFile, GoFileBinary, GoFileCode, GoFileDiff, GoFileDirectory, GoFileDirectoryFill, GoFileMedia, GoFileSubmodule, GoFileSymlinkFile, GoFileZip, GoFilter, GoFlame, GoFold, GoFoldDown, GoFoldUp, GoGear, GoGift, GoGitBranch, GoGitCommit, GoGitCompare, GoGitMerge, GoGitMergeQueue, GoGitPullRequest, GoGitPullRequestClosed, GoGitPullRequestDraft, GoGlobe, GoGoal, GoGrabber, GoGraph, GoHash, GoHeading, GoHeart, GoHeartFill, GoHistory, GoHome, GoHomeFill, GoHorizontalRule, GoHourglass, GoHubot, GoImage, GoInbox, GoInfinity, GoInfo, GoIssueClosed, GoIssueDraft, GoIssueOpened, GoIssueReopened, GoIssueTrackedBy, GoIssueTracks, GoItalic, GoIterations, GoKebabHorizontal, GoKey, GoLaw, GoLightBulb, GoLink, GoLinkExternal, GoListOrdered, GoListUnordered, GoLocation, GoLock, GoLog, GoMail, GoMegaphone, GoMention, GoMilestone, GoMirror, GoMoon, GoMortarBoard, GoMoveToBottom, GoMoveToEnd, GoMoveToStart, GoMoveToTop, GoMultiSelect, GoMute, GoNoEntry, GoNorthStar, GoNote, GoNumber, GoOrganization, GoPackage, GoPackageDependencies, GoPackageDependents, GoPaperAirplane, GoPaperclip, GoPasskeyFill, GoPaste, GoPencil, GoPeople, GoPerson, GoPersonAdd, GoPersonFill, GoPin, GoPlay, GoPlug, GoPlus, GoPlusCircle, GoProject, GoProjectRoadmap, GoProjectSymlink, GoProjectTemplate, GoPulse, GoQuestion, GoQuote, GoRead, GoRelFilePath, GoReply, GoRepo, GoRepoForked, GoRepoLocked, GoRepoPush, GoRepoTemplate, GoReport, GoRocket, GoRows, GoRss, GoRuby, GoScreenFull, GoScreenNormal, GoSearch, GoServer, GoShare, GoShareAndroid, GoShield, GoShieldCheck, GoShieldLock, GoShieldSlash, GoShieldX, GoSidebarCollapse, GoSidebarExpand, GoSignIn, GoSignOut, GoSingleSelect, GoSkip, GoSkipFill, GoSmiley, GoSortAsc, GoSortDesc, GoSponsorTiers, GoSquare, GoSquareFill, GoSquirrel, GoStack, GoStar, GoStarFill, GoStop, GoStopwatch, GoStrikethrough, GoSun, GoSync, GoTab, GoTable, GoTag, GoTasklist, GoTelescope, GoTelescopeFill, GoTerminal, GoThumbsdown, GoThumbsup, GoTools, GoTrash, GoTriangleDown, GoTriangleLeft, GoTriangleRight, GoTriangleUp, GoTrophy, GoTypography, GoUnfold, GoUnlink, GoUnlock, GoUnmute, GoUnread, GoUnverified, GoUpload, GoVerified, GoVersions, GoVideo, GoWorkflow, GoX, GoXCircle, GoXCircleFill, GoZap, GoZoomIn, GoZoomOut)

**Comment by rluders** [comment]
Thanks, @VishalSingh4310 for reporting it. Package `react-icons/go` seems to have removed the `GoThreeBars`. 😮‍💨 

Let's consider removing the dependency here. Now sure yet how, but... the [flowbite-icons](https://github.com/themesberg/flowbite-icons) isn't released yet. Maybe we just need to replace this icon at the component.

The weird thing is that it didn't "hit" any problems with our CI or even NextJS doc app. Maybe you are importing the newest `react-icons/go` and overwriting the `flowbite-react` dependency? 🤔 We use the `"react-icons": "^4.8.0",` check if the version matches for you, and if you are not importing the most recent one.

**Comment by VishalSingh4310** [comment]
@rluders I am not using `react-icons` anywhere in my project still getting issue for it

**Comment by tulup-conner** [comment]
I'm curious what's happening here because I can't reproduce this myself, but you are correct that GoThreeBars is gone. I would just use Font Awesome's `FaBars` instead. But when I build flowbite-next-starter with latest flowbite-react and react-icons, which has several Navbar.Toggles, I can build and serve and the hamburger menu works.

**Comment by tulup-conner** [comment]
What's happened here is that you have a newer version of react-icons than we are officially supporting. I will release a fix for this but I can see immediately upon upgrading the library to latest react-icons that this happens suddenly. I will upgrade us anyway since people are likely to have this happen

**Comment by VishalSingh4310** [comment]
@tulup-conner Thanks. That will be great help.

**Comment by AndrejGajdos** [comment]
please can you release the fix?

**Comment by phantomdenkali** [comment]
I'm running into this problem again. Is there a specific version I should be using of something?

 "tailwindcss": "2.2.16"
    "flowbite": "1.6.4",
    "flowbite-react": "0.4.9",

**Comment by arewageek** [comment]
I'm experiencing same issues here. Funny thing  is I'm not even using react icon in my project. Is there a way I can fix this?

**Comment by dimaslanjaka** [comment]
me too. iam also already installed `"react-icons": "^4.7.1"`, but still caught error below

```log
WARNING in ./node_modules/flowbite-react/lib/esm/components/Navbar/NavbarToggle.js 7:50-61
export 'GoThreeBars' (imported as 'GoThreeBars') was not found in 'react-icons/go' (possible exports: GoAlert, GoAlertFill, GoArchive, GoArrowBoth, GoArrowDown, GoArrowDownLeft, GoArrowDownRight, GoArrowLeft, GoArrowRight, GoArrowSwitch, GoArrowUp, GoArrowUpLeft, GoArrowUpRight, GoBeaker, GoBell, GoBellFill, GoBellSlash, GoBlocked, GoBold, GoBook, GoBookmark, GoBookmarkFill, GoBookmarkSlash, GoBookmarkSlashFill, GoBriefcase, GoBroadcast, GoBrowser, GoBug, GoCalendar, GoCheck, GoCheckCircle, GoCheckCircleFill, GoCheckbox, GoChecklist, GoChevronDown, GoChevronLeft, GoChevronRight, GoChevronUp, GoCircle, GoCircleSlash, GoClock, GoClockFill, GoCloud, GoCloudOffline, GoCode, GoCodeOfConduct, GoCodeReview, GoCodeSquare, GoCodescan, GoCodescanCheckmark, GoCodespaces, GoColumns, GoCommandPalette, GoComment, GoCommentDiscussion, GoCommit, GoContainer, GoCopilot, GoCopy, GoCpu, GoCreditCard, GoCrossReference, GoDash, GoDatabase, GoDependabot, GoDesktopDownload, GoDeviceCameraVideo, GoDeviceDesktop, GoDeviceMobile, GoDiamond, GoDiff, GoDiscussionClosed, GoDiscussionDuplicate, GoDiscussionOutdated, GoDot, GoDotFill, GoDownload, GoDuplicate, GoEye, GoEyeClosed, GoFile, GoFileBinary, GoFileCode, GoFileDiff, GoFileDirectory, GoFileDirectoryFill, GoFileMedia, GoFileSubmodule, GoFileSymlinkFile, GoFileZip, GoFilter, GoFlame, GoFold, GoFoldDown, GoFoldUp, GoGear, GoGift, GoGitBranch, GoGitCommit, GoGitCompare, GoGitMerge, GoGitMergeQueue, GoGitPullRequest, GoGitPullRequestClosed, GoGitPullRequestDraft, GoGlobe, GoGoal, GoGrabber, GoGraph, GoHash, GoHeading, GoHeart, GoHeartFill, GoHistory, GoHome, GoHomeFill, GoHorizontalRule, GoHourglass, GoHubot, GoImage, GoInbox, GoInfinity, GoInfo, GoIssueClosed, GoIssueDraft, GoIssueOpened, GoIssueReopened, GoIssueTrackedBy, GoIssueTracks, GoItalic, GoIterations, GoKebabHorizontal, GoKey, GoLaw, GoLightBulb, GoLink, GoLinkExternal, GoListOrdered, GoListUnordered, GoLocation, GoLock, GoLog, GoMail, GoMegaphone, GoMention, GoMilestone, GoMirror, GoMoon, GoMortarBoard, GoMoveToBottom, GoMoveToEnd, GoMoveToStart, GoMoveToTop, GoMultiSelect, GoMute, GoNoEntry, GoNorthStar, GoNote, GoNumber, GoOrganization, GoPackage, GoPackageDependencies, GoPackageDependents, GoPaperAirplane, GoPaperclip, GoPasskeyFill, GoPaste, GoPencil, GoPeople, GoPerson, GoPersonAdd, GoPersonFill, GoPin, GoPlay, GoPlug, GoPlus, GoPlusCircle, GoProject, GoProjectRoadmap, GoProjectSymlink, GoProjectTemplate, GoPulse, GoQuestion, GoQuote, GoRead, GoRelFilePath, GoReply, GoRepo, GoRepoForked, GoRepoLocked, GoRepoPush, GoRepoTemplate, GoReport, GoRocket, GoRows, GoRss, GoRuby, GoScreenFull, GoScreenNormal, GoSearch, GoServer, GoShare, GoShareAndroid, GoShield, GoShieldCheck, GoShieldLock, GoShieldSlash, GoShieldX, GoSidebarCollapse, GoSidebarExpand, GoSignIn, GoSignOut, GoSingleSelect, GoSkip, GoSkipFill, GoSmiley, GoSortAsc, GoSortDesc, GoSponsorTiers, GoSquare, GoSquareFill, GoSquirrel, GoStack, GoStar, GoStarFill, GoStop, GoStopwatch, GoStrikethrough, GoSun, GoSync, GoTab, GoTable, GoTag, GoTasklist, GoTelescope, GoTelescopeFill, GoTerminal, GoThumbsdown, GoThumbsup, GoTools, GoTrash, GoTriangleDown, GoTriangleLeft, GoTriangleRight, GoTriangleUp, GoTrophy, GoTypography, GoUnfold, GoUnlink, GoUnlock, GoUnmute, GoUnread, GoUnverified, GoUpload, GoVerified, GoVersions, GoVideo, GoWorkflow, GoX, GoXCircle, GoXCircleFill, GoZap, GoZoomIn, GoZoomOut)
```

**Comment by dimaslanjaka** [comment]
`GoThreeBars` not exported on `react-icons/go`

![Screenshot from 2023-10-11 14-54-24](https://github.com/themesberg/flowbite-react/assets/12471057/b71727b4-3730-4950-b59b-f99e63582347)

**Comment by johnkoshy** [comment]
1. I tried to import { GoPrimitiveDot } from 'react-icons/go' ;
2. I got this error:
export 'GoPrimitiveDot' (imported as 'GoPrimitiveDot') was not found in 'react-icons/go'
note: I already installed react-icons
3. I visited this page:
https://react-icons.github.io/react-icons
4. search for dot and choose a dot that best fit for your project
5. paste this in your project:
import { GoDotFill } from 'react-icons/go' ;
6. add the dot where you wanted in your project like this:
<span><GoDotFill /></span>
